src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchbar" data-qa="big">
+      <img
+        src="images/Search.svg"
+        alt="search-icon"
```

Comment:
no need to write icon or image words in alt, screenreaders know this is an image